packages/function-entry/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@pie-ui/function-entry",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "@pie-framework/pie-player-events": "^0.1.0",
+    "@pie-lib/editable-html": "^2.2.4",
+    "@pie-lib/render-ui": "^0.1.13",
+    "material-ui": "^1.0.0-beta.27",
+    "prop-types": "^15.6.0",
+    "react": "^16.2.0",
```

Comment:
Use react@^15.6.1 (and react-dom) for now, we need to co-ordinate the react upgrade.